### PR TITLE
Ci/CD and codeowners

### PR DESCRIPTION
**Pull request by afriemann**

### CODE REVIEW

File: .jenkins/python.yaml [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+spec:
+  containers:
+    - name: python
+      image: python:3-alpine
+      tty: true
+      command:
+        - cat
+      resources:
+        requests:
+          memory: '512Mi'
```
Comment (kernelold) [inline .jenkins/python.yaml]: does it really need so much?

File: Jenkinsfile [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+    stage('setup') {
+      steps {
+        sh 'apk add git gcc musl-dev linux-headers libffi-dev libressl-dev'
+        sh 'pip install --upgrade tox'
+      }
+    }
+
+    stage('test') {
+      steps {
+        sh 'tox'
```
Comment (kernelold) [inline Jenkinsfile:29]: tox.ini missing(I cant see it)?

**Review by kernelold — DISMISSED** [review]

**Comment by kernelold** [comment]
in general looks ok, just a few questions

File: Jenkinsfile (hunk 2 above)
Comment (afriemann) [reply Jenkinsfile:29]: already there: https://github.com/ClarkSource/aws-exporter/blob/master/tox.ini

File: .jenkins/python.yaml (hunk 1 above)
Comment (afriemann) [reply .jenkins/python.yaml]: probably not, I can reduce it

**Review by kernelold — APPROVED** [review]
👍

File: .jenkins/python.yaml [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+      image: python:3-alpine
+      tty: true
+      command:
+        - cat
+      resources:
+        requests:
+          memory: '256Mi'
+          cpu: '100m'
+        limits:
+          memory: '256Mi'
```
Comment (kernelold) [inline .jenkins/python.yaml:17]: limits:
I think we can keep 512 in case of OOM... 256 should be fine I guess

**Pull request merged**